### ISSUE TITLE
Post notifications not sent on re-published posts

### ISSUE THREAD

**Issue by metabreakr** [Automattic/wp-calypso#53800]
For users who wish to evergreen content by un-publishing a post, and scheduling it to publish on a new date, post notifications are not sent to subscribed users, as technically the notification already went out upon first publish.

### Steps to reproduce the behavior

1. Publish a post today
2. Observe that a post notification is sent
3. Unpublish that post tomorrow, and schedule it to publish again the day after
4. Once published again, no notification is sent

#### What I expected to happen

While it's logical for a post notification not to be sent a second (or third, fourth, etc.) time, this doesn't serve users who wish to evergreen content. I would expect a method in which a notification was sent again. 

#### What actually happened

Besides the first, no further notifications are sent. 

### Context

4058978-zen

**Comment by kwight** [comment]
This is intended behaviour (if not optimal for everyone), but I've kept it open as a feature request.

**Comment by jordesign** [comment]
If we were considering work on https://github.com/Automattic/wp-calypso/issues/44263 - a change to the default behaviour on re-publishing posts could be a good step.

Then the user opts in/out on sending the notifications at the re-publishing stage as well.

**Comment by autumnfjeld** [comment]
@inaikem  Another ticket related to subscriptions, just wanted to check if this is better done in the Subscriptions pod or if we in the Blogger Pod are ok to work on this.

**Comment by inaikem** [comment]
I'll bow to @kwight to confirm if we'll be able to deal with this as a feature request in the pod. Thanks Autumn, keep them coming 🙏

**Comment by kwight** [comment]
No, I don't see this being high enough priority for us – go for it @autumnfjeld !
